src/constants/constants.js

```diff
@@ -0,0 +1,9 @@
+export const ERROR_MESSAGES = {
+	INVALID_TYPE: '잘못된 타입입니다.',
+	INVALID_INPUT: '잘못된 입력입니다.',
+	NEGATIVE_NUMBER: '음수는 입력할 수 없습니다.',
+	INVALID_NUMBER: '양의 정수가 아닙니다.',
+	INVALID_SEPARATER: '잘못된 구분자입니다.',
+};
```

Comment:
```suggestion
export const ERROR_MESSAGES =  Object.freeeze({
	INVALID_TYPE: '잘못된 타입입니다.',
	INVALID_INPUT: '잘못된 입력입니다.',
	NEGATIVE_NUMBER: '음수는 입력할 수 없습니다.',
	INVALID_NUMBER: '양의 정수가 아닙니다.',
	INVALID_SEPARATER: '잘못된 구분자입니다.',
});
```
Object.freeze를 사용해서 상수를 불변하게 관리 하면 좋을 것 같요!!
추후 변동 사항이 있더라도 수정을 더욱 간편하게 한다는 목적을 가지고 상수화를 하는 것이기 때문에,
에러메세지 뿐만아니라 여러가지 조건이나 인덱스 입출력 문 같은경우에도 상수화를 해보는 것도 좋을 것 같습니다